src/Microsoft.Health.Dicom.Core/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.Extensions.DependencyInjection;
+
+namespace Microsoft.Health.Dicom.Core.Extensions
+{
+    public static class IServiceCollectionExtensions
```

Comment:
They don't add a lot of value, do we really need these wrapper methods?

Reply:
I feel it will, so far as I know these are required:
1. Core layer
   * AddExtendedQueryTagService
   * DicomTagParser
   * ExtendedQueryTagEntryValidator
2. Sql Layer
   * SqlInstanceStore
   * SqlExtendedQueryTagStoreV1
   * SqlExtendedQueryTagStoreV2
   * SqlExtendedQueryTagStoreV3
   * SqlStoreFactory<ISqlExtendedQueryTagStore, IExtendedQueryTagStore>>
   * SqlReindexStore

Once we add support to log exception, probably need components to access table store. 
I think the existing pattern for DicomServer is good, and following that could have a easy understanding code.